server/spec/hypervisor_check_in_spec.rb

```diff
@@ -1201,18 +1208,10 @@ def create_virtwho_client(user)
         ],
         "facts" => {"test_fact" => "fact_value" }
     }
-    job_detail = send_host_guest_mapping(owner, user, report.to_json())
-    job_detail["state"].should == "FAILED"
-
-    # empty json
-    report = {}
-    job_detail = send_host_guest_mapping(owner, user, report.to_json())
-    job_detail["state"].should == "FAILED"
-
-    # this is the correct version of an empy list of hypervisors
-    report = {hypervisors:[]}
-    job_detail = send_host_guest_mapping(owner, user, report.to_json())
-    job_detail["state"].should == "FINISHED"
+    virtwho = create_virtwho_client(@user)
+    lambda do
+      virtwho.hypervisor_update(@owner['key'], report.to_json())
+    end.should raise_exception(RestClient::BadRequest)
```

Comment:
Replacing the first assertion with this is ok, but we should probably not remove the other 2 assertions (with report `{}` and `{hypervisors:[]}`.

Reply:
Done.